### PR TITLE
Branch kk filter by status

### PR DESCRIPTION
**Pull request by Ko-Khan**
-  Add new command set -a
- With this new command, user can now **set** up a filter based on status of an article.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SetCommand.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package seedu.address.logic.commands;
+
+/**
+ * Sets predicates that filter article book
+ */
+public class SetCommand {
```
Comment (Murugan-Maniish) [inline src/main/java/seedu/address/logic/commands/SetCommand.java:6]: Why not "FilterCommand"

File: src/main/java/seedu/address/logic/commands/articlecommands/SetArticleCommand.java
Comment (Murugan-Maniish) [inline src/main/java/seedu/address/logic/commands/articlecommands/SetArticleCommand.java:1]: LGTM

**Review by Murugan-Maniish — APPROVED** [review]
Overall LGTM, nothing looks off.

File: src/main/java/seedu/address/logic/commands/SetCommand.java (hunk 1 above)
Comment (Ko-Khan) [reply src/main/java/seedu/address/logic/commands/SetCommand.java:6]: I wanted to emphasize that using the command **sets** the filter for all future seraches.

**Pull request merged**